uPortal-webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -90,15 +90,15 @@
         <filter-class>org.jasig.cas.client.validation.Cas20ProxyReceivingTicketValidationFilter</filter-class>
         <init-param>
           <param-name>casServerUrlPrefix</param-name>
-          <param-value>${environment.build.cas.protocol}://${environment.build.cas.server}${environment.build.cas.context}</param-value>
+          <param-value>http://localhost:8080/cas</param-value>
```

Comment:
`Cas20ProxyReceivingTicketValidationFilter` could be enhanced to accept configuration via environment variables preferentially over its configuration via filter init-params. 

Or using Spring technology put a delegating filter in web.xml that delegates to a Spring bean wiring up the `Cas20ProxyReceivingTicketValidationFilter` instance, with that bean then configured in the preferred way via Spring.